src/content/docs/ko/guides/integrations-guide/vue.mdx

```diff
@@ -0,0 +1,192 @@
+githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/vue/'
+category: renderer
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+**[Astro 통합](/ko/guides/integrations-guide/)** 은 프로젝트에 [Vue 3](https://vuejs.org/) 컴포넌트를 통해 서버 측 랜더링(SSR)과 클라이언트 랜더링(CSR)에서의 하이드레이션을 가능하게 합니다.
+
+## 설치방법
+
+Astro에서는 공식 통합설정을 자동으로 진행할 수 있는 `astro add` 명령이 포함되어 있습니다. 원할경우, [수동으로 설치](#수동-설치) 할 수 있습니다.
+
+`@astrojs/vue` 를 설치하려면, 프로젝트 폴더에서 아래의 커맨드를 실행하고 프롬프트를 따라줍니다:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npx astro add vue
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm astro add vue
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn astro add vue
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+만일 문제가 발생할 경우, [GitHub에 문제 리포트](https://github.com/withastro/astro/issues) 하고 아래의 수동설치를 시도합니다.
+
+### 수동 설치
+
+우선, `@astrojs/vue` 패키지를 설치합니다:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npm install @astrojs/vue
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm add @astrojs/vue
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn add @astrojs/vue
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+대부분의 패키지 매니저는 관련된 피어 종속성도 설치합니다. Astro를 실행 시 "'vue' 패키지를 찾을 수 없습니다"(또는 이와 유사한 경고)가 표시되면, Vue를 설치해야 합니다:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npm install vue
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm add vue
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn add vue
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+그런 다음, `integrations` 속성을 사용하여 `astro.config.*` 파일에 통합적용 해줍니다:
+
+```js ins={2} ins="vue()" title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  // ...
+  integrations: [vue()],
+});
+```
+
+## 시작하기
+
+Astro에 첫 번째 Vue 컴포넌트를 사용하려면,  [UI 프레임워크 문서](/ko/core-concepts/framework-components/#using-framework-components) 로 이동해줍니다. 이 문서에서 아래와 같은 내용을 확인할 수 있습니다:
+
+* 📦 프레임워크 컴포넌트가 로드되는 방식,
+* 💧 클라이언트 측(client-side) 하이드레이션 옵션 및
+* 🤝 프레임워크를 함께 사용하고 혼합 사용할 수 있는 기회
+
+## 문제 해결
+
+도움이 필요할 경우, [Discord](https://astro.build/chat) 의 `#support` 채널을 확인해줍니다. 우리의 친절한 지원팀 멤버들이 도와드립니다!
+
+또한 [Astro 통합 문서](/ko/guides/integrations-guide/) 를 통해, 더 많은 통합과정에 대한 내용을 확인합니다.
+
+## 기여
+
+이 패키지는 Astro의 코어 팀이 유지관리 합니다. 문제가 PR은 언제든 환영합니다!
+
+## 옵션
+
+이 통합은 `@vitejs/plugin-vue` 의 도움을 받습니다. Vue 컴파일러를 커스터마이징 하기 위해서는, 통합에 옵션을 제공할 수 있습니다. 자세한 사항은 `@vitejs/plugin-vue` [문서](https://www.npmjs.com/package/@vitejs/plugin-vue) 를 참조합니다.
+
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  // ...
+  integrations: [
+    vue({
+      template: {
+        compilerOptions: {
+          // treat any tag that starts with ion- as custom elements
```

Comment:
Can you translate this comment?

Reply:
ah! I missed annotation section!
I,ll translate it!🙇🏻‍♀️🙇🏻‍♀️